content/post/3Drendering.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
I think this needs a rewrite, focussing just on aesthetics. Leave the interactive github action view as an aside at the end.
include some images of your favourite maximo renders.. check his twitter out to get some images: https://twitter.com/maxiborga

don't link to discord (it's private). 

video was just by maximo . proppy not involved

use my inverter, but include something liek 'by following along on the video I was able to render my first ever GDS - an inverter I drew back in 2000. link this vid: https://www.youtube.com/watch?v=IQ_DcWT_cbc